src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -496,6 +496,34 @@ tests:
     - label:
           "Step 8a: TH sends a AddScene command to DUT with the GroupID field
           set to G1, the SceneID field set to 0x01, the TransitionTime field set
+          to 60 000 000 (60 000s) and no extension field sets. This should fail and return
+          a status of 0x85 (INVALID_COMMAND) because we exceed the max TransitionTime"
+      PICS: S.S.C00.Rsp && PICS_SDK_CI_ONLY
+      command: "AddScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: G1
+              - name: "SceneID"
+                value: 0x01
+              - name: "TransitionTime"
+                value: 60000000
```

Comment:
I suggest a number just over 6000s.  For example, 7000000.

Reply:
Applied